### PR TITLE
moment js

### PR DESCRIPTION
**Pull request by alltrust**
imported moment js, updated properties with created time and due date. These are formatted with moment.js library

### CODE REVIEW

File: src/components/CreateReminders/CreateReminder.js [hunk 1]
```diff
@@ -75,19 +76,18 @@ const CreateReminders = () => {
...
     const reminderNotes = notesRef.current.value;
 
-    let today = new Date();
-    const dd = String(today.getDate()).padStart(2, "0");
-    const mm = String(today.getMonth() + 1).padStart(2, "0");
-    const yyyy = today.getFullYear();
-    today = yyyy + "-" + mm + "-" + dd;
+    const dueDate = moment(reminderDueDate).format("YYYY MMMM dddd")
+
+    let calendarNow = moment().calendar()
```
Comment (sebastianserrano) [inline src/components/CreateReminders/CreateReminder.js:81]: How about just `now`? also, can't this just be `moment()`?

File: src/components/CreateReminders/CreateReminder.js [hunk 2]
```diff
@@ -75,19 +76,18 @@ const CreateReminders = () => {
...
+    console.log(calendarNow)
```
Comment (sebastianserrano) [inline src/components/CreateReminders/CreateReminder.js:82]: Why keep this?

**Comment by sebastianserrano** [comment]
Reference ticket in the description

File: src/components/CreateReminders/CreateReminder.js [hunk 3]
```diff
@@ -75,19 +75,16 @@ const CreateReminders = () => {
     const reminderDueDate = dateInputRef.current.value;
     const reminderName = nameInputRef.current.value;
     const reminderNotes = notesRef.current.value;
-
-    const dueDate = moment(reminderDueDate).format("YYYY MMMM dddd")
-
-    let calendarNow = moment().calendar()
-    console.log(calendarNow)
+    const dueDate = moment(reminderDueDate).format("MMM Do YY")
```
Comment (alltrust) [inline src/components/CreateReminders/CreateReminder.js:78]: @sebastianserrano  updated and removed console.logs

File: src/components/CreateReminders/CreateReminder.js (hunk 2 above)
Comment (alltrust) [reply src/components/CreateReminders/CreateReminder.js]: changed to 

```javscript
let now = moment().format("MMM Do YY")

```

File: src/components/CreateReminders/CreateReminder.js (hunk 3 above)
Comment (sebastianserrano) [reply src/components/CreateReminders/CreateReminder.js:78]: Sounds good

**Pull request merged**